docker/Dockerfile

```diff
@@ -0,0 +1,63 @@
+# Dockerfile for building the Moq Relay Server
+#
+# docker build -t moqrelay -f docker/Dockerfile .
+# 
+# To run the container:
+#
+# Using default values (port 4433, logging DBG)
+# docker run --rm -p 4433:4433 moqrelay
+#
+# Override logging level
+# docker run --rm -p 4433:4433 -e MOQ_LOG_LEVEL=INFO moqrelay
```

Comment:
It may be helpful (for non-Meta folks) to include a description of how this project uses the [available log levels](https://github.com/facebook/folly/blob/main/folly/logging/docs/LogLevels.md).

Reply:
it looks this is folly XLOG and the levels are: ERR, WARN, INFO, and DBG1-6 (for increasing levels of trace) . however I can't see the '--logging' flag having any effect. it is possible to set the following env vars and get richer out put:

GLOG_minloglevel=0 GLOG_v=6 GLOG_logtostderr=1 ./_build/bin/moqrelayserver -port 4433 -cert ./certs/certificate.pem -key ./certs/certificate.key -endpoint "/moq"

@afrind am i seeing this right - suspect some logging init/flag processing code is needed?